go-apps/meep-loc-serv/sbi/loc-serv-sbi.go

```diff
@@ -172,8 +180,15 @@ func processActiveScenarioUpdate() {
 	poaMap, _ := sbi.pc.GetAllPoa()
 
 	// Update UE info
+	ueNames := []string{}
 	ueNameList := sbi.activeModel.GetNodeNames("UE")
 	for _, name := range ueNameList {
+		// Ignore disconnected UEs
+		if !isUeConnected(name) {
```

Comment:
Question - 
Instead of adding `if !isUeConnected(name)` -- wouldn't it be simpler to take a scenario object and delete all disconnected UEs... then pass this cleaned scenario as an input to the service?

Reply:
The current callback API between the SBI & location service updates a single UE at a time. The SBI must also inform the location service when a UE has been removed or disconnected by setting empty strings in the callback function. This could lead to temporarily invalid location service states.

In order to fix this we should rework the SBI/location service callback API. Your suggestion then applies for disconnected node processing. This API rework should be done but will not be addressed in the scope of this PR.